### DISCUSSION TITLE
Custom Application Integration with YetiForce

### DISCUSSION THREAD

**Discussion by drakub** [YetiForceCompany/YetiForceCRM#14393] [General]
Is there any possibility of having a custom application developed independent as a portal and allow it to integrate with YetiForce CRM through APIs?

**Comment by mariuszkrzaczkowski** [comment]
it's open source you can do anything

**Reply by mariuszkrzaczkowski** [reply]
https://doc.yetiforce.com/ 
https://github.com/YetiForceCompany/YetiForceScripts/tree/master/YetiREST
